crates/uv-interpreter/src/interpreter.rs

```diff
@@ -287,28 +304,97 @@ impl Interpreter {
     pub fn implementation_name(&self) -> &str {
         &self.markers.implementation_name
     }
+
+    /// Return the `sys.base_exec_prefix` path for this Python interpreter.
     pub fn base_exec_prefix(&self) -> &Path {
         &self.base_exec_prefix
     }
+
+    /// Return the `sys.base_prefix` path for this Python interpreter.
     pub fn base_prefix(&self) -> &Path {
         &self.base_prefix
     }
 
-    /// `sysconfig.get_path("stdlib")`
-    pub fn stdlib(&self) -> &Path {
-        &self.stdlib
-    }
+    /// Return the `sys.executable` path for this Python interpreter.
     pub fn sys_executable(&self) -> &Path {
         &self.sys_executable
     }
+
+    /// Return the `purelib` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn purelib(&self) -> &Path {
+        &self.sysconfig.purelib
+    }
+
+    /// Return the `platlib` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn platlib(&self) -> &Path {
+        &self.sysconfig.platlib
+    }
+
+    /// Return the `scripts` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn scripts(&self) -> &Path {
+        &self.sysconfig.scripts
+    }
+
+    /// Return the `data` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn data(&self) -> &Path {
+        &self.sysconfig.data
+    }
+
+    /// Return the `include` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn include(&self) -> &Path {
+        &self.sysconfig.include
+    }
+
+    /// Return the `sysconfig` paths for this Python interpreter.
+    pub fn sysconfig(&self) -> &Sysconfig {
+        &self.sysconfig
+    }
+
+    /// Return the [`Target`] environment used to install wheels into this interpreter.
+    pub fn target(&self) -> Target {
+        Target {
+            python_version: self.python_tuple(),
+            sys_executable: self.sys_executable().to_path_buf(),
+            purelib: self.purelib().to_path_buf(),
+            platlib: self.platlib().to_path_buf(),
+            scripts: self.scripts().to_path_buf(),
+            data: self.data().to_path_buf(),
+            // Detect if this is a PEP 405-compliant virtual environment.
+            // See: https://github.com/pypa/pip/blob/0ad4c94be74cc24874c6feb5bb3c2152c398a18e/src/pip/_internal/utils/virtualenv.py#L14.
+            include: if self.prefix == self.base_prefix {
+                // If the interpreter is a venv, then the `include` directory has a different structure.
+                // See: https://github.com/pypa/pip/blob/0ad4c94be74cc24874c6feb5bb3c2152c398a18e/src/pip/_internal/locations/_sysconfig.py#L172
+                self.prefix.join("include").join("site").join(format!(
+                    "python{}.{}",
+                    self.python_major(),
+                    self.python_minor()
+                ))
```

Comment:
\cc @konstin - But when installing, there's custom logic. (We had this in `install-wheel-rs`, which prior to this PR assumed a virtualenv.)

Reply:
If you look at the linked pip source, they also have this logic around `user` and `home`. Is that significant here?

Reply:
I don't think i ever had to deal with `include`, sorry, no idea